engine/src/main/java/com/redhat/cloud/notifications/db/repositories/EndpointRepository.java

```diff
@@ -333,6 +336,42 @@ public boolean disableEndpoint(Endpoint endpoint) {
         return updated > 0;
     }
 
+    /**
+     * Finds all the integration names of the given type and groups them by
+     * the organization identifier.
+     * @param integrationType the type of the integration we want to find.
+     * @return a map with the organization ID as the key, and a list of
+     * integration names as the value.
+     */
+    public Map<String, List<String>> findIntegrationNamesByTypeGroupedByOrganizationId(final CompositeEndpointType integrationType) {
+        final String findByTypeQuery =
```

Comment:
Shouldn't we restrict results to only used teams integrations (linked to an event type) and also to those without the new teams url pattern?

Reply:
We're not 100% sure about the new URL pattern, so it's risky to exclude some integrations based on a guess.

Reply:
About the used/unused integrations, that's probably a question for PM.

Reply:
This was addressed in Slack. Resolving.